### DISCUSSION TITLE
Make all uploads use at least one common format

### DISCUSSION THREAD

**Discussion by lukasmueller** [solgenomics/sgn#3447] [General]
Expected Behavior 
--------------------------------------------------------------------------

Should all uploads offer at least a csv options? Or tab delimited? Needs discussion...

For Bugs:
---------

**Comment by ch728** [comment]
I think this would be helpful. It has been a source of confusion for some of the users in the program I work with. Tab-delimited is not ideal because it requires extra steps to import into Excel and csv would cause issues with the accession file uploads, which have comma separated values in some fields. As much as I dislike Excel and other binary file formats, having a .xls upload across all tools might be the best way go for user friendliness.

**Comment by lukasmueller** [comment]
I think with the new upload module we can close this discussion.
